rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -65,120 +100,100 @@ valuetype MyValueType2 {
+### Packages: bundling elements to a package for decoupled usage
 
-For bundling and publishing elements, the RFC introduces a new concept called `libraries`.
-A `library` can inhibit `Valuetype`s, `Block`s, `BlockType`s, `Constraint`s, and `Transform`s.
-A library is published per default.
-All elements within a library have to use the `publish` keyword.
+For bundling and publishing elements, the RFC introduces a new concept called `packages`.
+A `package` can inhibit `Valuetype`s, `Block`s, `BlockType`s, `Constraint`s, `Transform`s, and further `Package`s.
+A package must be of visibility `file-published` and, thus, requires the keyword `publish`.
+Elements within a package can use the `publish` keyword to make them of visibility `package-published` or are `package-private` per default.
 
-**Example library**
+**Example package**
 
 ```
-library MyDomainLibrary {
-  // definition of a new valuetype as part of the library
+publish package MyDomainPackage {
+  // definition of a new valuetype as part of the package
   publish valuetype MyDomainSpecificValuetype1 {
     // ... details of valuetype
   }
 
-  // reference to an existing valuetype to make it part of the library
+  // reference to an existing valuetype to make it part of the package
   publish MyDomainSpecificValuetype2;
 
   // ... possibly more elements
 }
 ```
 
-### Visibility of elements in a file
-
-By introducing the concept of `libraries`, most elements can be defined on three levels in a Jayvee file:
-
-1. On the root level of the file
-2. Within a pipeline
-3. (new) Within a library
-
-The name of an element is given by its definition.
-The **qualified name** is constructed by prepending container structures in this pattern: `<container name>.<element name>`, e.g., `MyDomainLibrary.MyDomainSpecificValuetype1`.
-
-**Access paths:**
-
-- root level elements can access
-  - root level elements by their name
-  - and elements of libraries by their qualified name
-- elements within a library can access
-  - root level elements by their name
-  - and elements of other libraries by their qualified name
-- elements within a pipeline can access
-  - root level elements by their name
-  - elements within the same pipeline by their name
-  - and elements of libraries by their qualified name
-
-**No-access paths:**
-
-- elements within a pipeline cannot be referenced by outside elements
-
-**Used elements of different files are handled as if they were defined at the root level.**
+The advantage of bundling elements into a `package` is the decoupling from the internal file system structure.
+Rather than accessing files directly (and needing knowledge what is element is located in which files) users can simply use a whole package with all its elements (and don't need to know in which file the element is originally defined).
 
 ### Using elements
 
-Only `publish`ed elements can be used in other files.
+Only `file-published` (with keyword `publish`) elements can be used in other files.
```

Comment:
I'd write `use`d to make it clear you are talking about importing using the use keyword, not just using in the general english sense.